astropy/cosmology/io/utils.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+from astropy.cosmology.parameter import Parameter
+from astropy.table import Column
+from astropy.modeling import Parameter as ModelParameter
+
+
+def convert_Parameter_to_Column(parameter, value, meta=None):
+    """Convert a Cosmology Parameter to a Table Column.
+
+    Parameters
+    ----------
+    parameter : `astropy.cosmology.parameter.Parameter`
+    value : Any
+    meta : dict or None, optional
+        Information from the Cosmology's metadata.
+
+    Returns
+    -------
+    `astropy.table.Column`
+    """
+    format = None if value is None else parameter.format_spec
+    shape = (1,) + np.shape(value)  # minimum of 1d
+
+    col = Column(data=np.reshape(value, shape),
+                 name=parameter.name,
+                 dtype=None,  # inferred from the data
+                 shape=shape,
+                 length=1,  # Cosmology is scalar
+                 description=parameter.__doc__,
+                 unit=None,  # inferred from the data
+                 format=format,
+                 meta=meta,
+                 copy=False,
+                 copy_indices=True)
```

Comment:
The last two are the same as the default, so I'd remove it. In the super-unlikely case that we ever decide the default should change, you'd probably want a change here too!

Reply:
Done.